src/Collection.php

```diff
@@ -882,6 +882,15 @@ public function insertMany(array $documents, array $options = [])
      */
     public function insertOne($document, array $options = [])
     {
+        if (isset($options['session']) && $options['session']->isInTransaction()) {
```

Comment:
This may need an `instanceof` check between `isset()` and calling `isInTransaction()`.